CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta8 (in progress)
+=========================
+
+New Features
+------------
+
+- You can now use the [Network Quality](https://www.twilio.com/docs/video/using-network-quality-api)
+  API to control the verbosity of the network quality statistics that are reported
+  for the LocalParticipant and RemoteParticipants in a Group Room. A Participant will
+  now have an additional property `networkQualityStats` which contains the network
+  quality statistics used to calculate the `networkQualityLevel`. (JSDK-2255)
```

Comment:
- Maybe [Network Quality API](https://www.twilio.com/docs/video/using-network-quality-api)
- It almost seems there are two unrelated things here: NQ API to control verbosity and an addition property on Participant. Are they related?
- Also, for new features, I think it's worth to introduce a section stating what this could be used for. e.g. With this API, you can create an network status overlay or alert the user of long lasting network issues causing a break in communication"

Reply:
@aymenn ,

> It almost seems there are two unrelated things here: NQ API to control verbosity and an addition property on Participant. Are they related?

Yes. By introducing verbosity levels, we are also introducing `networkQualityStats` which houses the statistics based on which the NQ level is calculated.

> Also, for new features, I think it's worth to introduce a section stating what this could be used for. e.g. With this API, you can create an network status overlay or alert the user of long lasting network issues causing a break in communication"

I agree, although I think the wagtail tutorial page for Network Quality (which already exists) is the place to do it.

Reply:
Should we reference that tutorial here?